doc/book/admin/troubleshoot.rst

```diff
@@ -314,3 +314,379 @@ recommend to optimize your Tarantool application code).
 
 If the value is greater than 0.01, your application definitely needs thorough
 code analysis aimed at optimizing memory usage.
+
+.. _admin-troubleshoot-finalizer_yielding:
+
+--------------------------------------------------------------------------------
+Problem: Fiber switch is forbidden in ``__gc`` metamethod
+--------------------------------------------------------------------------------
+
+~~~~~~~~~~~~~~~~~~~~~~~~
+Problem description
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Fiber switch is forbidden in ``__gc`` metamethod since `this change <https://github.com/tarantool/tarantool/issues/4518#issuecomment-704259323>`_
+to avoid unexpected Lua OOM.
```

Comment:
Well, I'm not sure, whether we need to describe this issue in a more precise way: on the one hand it looks making everything clearer for "experts"; on the other hand it might sound like some kind of elvish and reveals the details Tarantool user doesn't have to think about. Let's leave it as is for now.

Reply:
Ok. If we decide to describe the issue in more details, let's do it in a separate doc ticket.